src/store/getters/state-getters.ts

```diff
@@ -503,15 +490,26 @@ export const hasConversionDataChanged = (state: StateIF): boolean => {
 /** Whether the subject correction filing is valid. */
 export const isCorrectionValid = (state: StateIF): boolean => {
   if (isBenCorrectionFiling(state)) {
-    return (
-      getFlagsCompanyInfo(state).isValidCompanyName &&
-      getFlagsCompanyInfo(state).isValidNameTranslation &&
-      getFlagsCompanyInfo(state).isValidOrgPersons &&
-      getFlagsCompanyInfo(state).isValidShareStructure &&
-      getFlagsReviewCertify(state).isValidDetailComment &&
-      getFlagsReviewCertify(state).isValidCertify &&
-      getFlagsReviewCertify(state).isValidStaffPayment
-    )
+    if (isClientErrorCorrection(state)) {
```

Comment:
I separated Ben corrections into staff and client similar to how firm corrections are.

Reply:
I think I already did this in yesterday's PR.

Reply:
Right I think you did, I haven't update my local in a day or two.